### PR TITLE
Update docs and links.

### PR DESCRIPTION
**Pull request by galachad**
Fix links and update small details in docs.

### CODE REVIEW

**Comment by ramiromagno** [comment]
Links are fixed now. I am guessing you will update the CI issues in a separate pull request?

**Review by ramiromagno — APPROVED** [review]
Links have been fixed. All good.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
-# Contribution to {oak} 
+# Contribution to {sdtm.oak} 
 
 This outlines how to propose a change to the aok package. For more detailed info
```
Comment (edgar-manukyan) [inline .github/CONTRIBUTING.md:3]:
```suggestion
This outlines how to propose a change to the {sdtm.oak} package. For more detailed info
```

**Review by edgar-manukyan — APPROVED** [review]
Thanks so much Adam, I left one suggestion for your kind consideration.

**Pull request merged**